playwright/models/OnboardingPage.ts

```diff
@@ -0,0 +1,40 @@
+import { Page } from '@playwright/test';
+
+export class OnboardingPage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async getStarted() {
+    await this.page.click('#onboarding-get-started-button');
+  }
+
+  async acceptTerms() {
+    await this.page.click("#onboarding-terms-check");
+    await Promise.all([
+      this.page.waitForResponse(/\.svg$/),
+      this.page.click('#onboarding-terms-submit'),
+    ]);
+  }
+
+  async selectDevice(device) {
+    await this.page.click(`button:has-text("${device}")`)
+  }
+
+  async connectDevice() {
+    await this.page.click(
+      'button:has-text("Connect deviceConnect your Nano XIs your device already set up? Connect it to th")',
```

Comment:
Seems like a bit of a flaky string. Probably ok for now but worth review what elements need better `id`s or `name`s in future

Reply:
^ 100% agreed

Reply:
Yes! You're right, I planned to rework those tests during our migration but why not now 😄 .

To be clear, we should not rely on elements text, because it's flaky ofc and each time the wording changes we'll have false negative errors. It'll also block us from testing languages on Ledger Live.

Moreover, `:has-text` is a pseudo-class that is not a part of [CSS3](https://www.w3.org/TR/selectors-3/) or even [CSS4](https://www.w3.org/TR/selectors-4/) specifications, those are [custom classes](https://playwright.dev/docs/selectors#css-selector) (_Playwright adds custom pseudo-classes like :visible, :text and more._) added by Playwright itself, so we must avoid to use them for the moment.